File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        int answer = 0;
        String change = "";
        for(int i = 0; i < myString.length(); i++){
            char c = myString.charAt(i);
            if(c == 'A'){
                change += 'B';
            }else{
                change += 'A';
            }
        }
        
        if(change.contains(pat)){
            answer = 1;
        }else{
            answer = 0;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.py
def solution(myString, pat):
    answer = 0
    myString = myString.replace('A','b')
    myString = myString.replace('B','a')
    
    pat = pat.replace('A','a')
    pat = pat.replace('B','b')

    if pat in myString:
        answer = 1
    return answer